views/session.hbs

```diff
@@ -54,7 +54,7 @@
       document.cookie = "jiraHost=" + window.jiraHost + "; path=/; max-age=" + 60 * 5 + "; samesite=none; secure";
       document.cookie = "jwt=" + window.jwt + "; path=/; max-age=" + 60 * 5 + "; samesite=none; secure";
     }
-    window.location = "{{redirectUrl}}";
+    window.location = decodeURIComponent("{{redirectUrl}}");
```

Comment:
Can you please explain why we first encoding the URL then decoding it before using it?

Reply:
You reading from the URL, it is already encoded. Here you're encoding in `session-get.ts` and decoding in `session.hbs` before navigating to the URL in browser.  This cycle of encoding and decoding seems to be redundant. Please verify it.

Reply:
The `=` gets converted, can see in the video below.

https://user-images.githubusercontent.com/13076255/182507147-685ec015-fb0e-4b7f-a56e-88071df0d47b.mov

Reply:
Ignore, found a better solution for this

Reply:
Tripe curly braces for HTML escaping. This is not meant for URL encoding.

Reply:
The aim is to correctly display the special characters(`=`), and for that [triple braces](https://handlebarsjs.com/guide/expressions.html#html-escaping) would be valid, isn't it?
What are you suggesting?

Reply:
I mixed `{{` with `{{{`, it is fine.